src/System Application/App/Email/src/Account/EmailAccounts.Page.al

```diff
@@ -402,7 +402,7 @@ page 8887 "Email Accounts"
         CurrPage.Update(false);
     end;
 
-    local procedure FilterToConnectorv2Accounts(var EmailAccounts: Record "Email Account")
+    local procedure FilterToConnectorv2v3Accounts(var EmailAccounts: Record "Email Account")
     var
         IConnector: Interface "Email Connector";
```

Comment:
can we please not add i to interfaces... we don't add c to codeunits nor t to tables...

Reply:
While I get where that is coming from, because it's true we don't prefix p for Params, nor CU to Codeunits, there might be some mangling of examples.

See:  https://youtu.be/PHmFqehrPG4?si=Kg1o5s-kiImOt1_k&t=638

Examples provided in the Launch events on the Interface include it.

But, it's also causing challenges with Prefix style affixes - before or after the I?  Both look weird.

Maybe there can be a nerdy as heck PGI "Interface prefix, to I or not to I?" or just a Yammer thread about it

Reply:
> can we please not add i to interfaces... we don't add c to codeunits nor t to tables...

It's very commonly used in DotNet. I would really like us to follow these naming conventions.
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces.

In short: add I as a prefix for Interfaces, but not C before classes.
![image](https://github.com/user-attachments/assets/5b2e5e43-e3a6-4f52-a2e9-089aa9c695e8)

Reply:
For the Object name, I agree that we shouldn't prefix it with `I` as we always know it is of type Interface when we have to define it. 

For the variable/parameter name, I have no particular preference for or against. But the preference I have is that we stay consistent throughout. Having it makes it clearer that it is an interface and that it is not of some other type, but what value + or - does it bring to just add it for interfaces? If we don't do it for tables, pages, codeunits.. Why do we start for interfaces besides similarity to how DotNet does?

Also, I don't think DotNet has a naming convention regarding the the variable/parameter name. It only has it for the object name.

```
interface ITest {...}
class Tester : ITest {...}

void SomeFunc()
{
    ITest test = new Tester();
}
```
As far as I can tell, from limited searching, the variable name wouldn't contain `I` nor any prefix to represent the variable as an Interface. 

I will leave this thread conversation open for now while I continue trying to fix my build failures :(

Reply:
My 2 cents: AL is a language onto its own. Yes we can peek and steal from other languages things that have proven 
 to work but we shouldn't do things simply because 'they' do it... 

The naming should convey meaning and add value. In my opinion, the i doesn't add either. When i call Foo.Bar() does it matter that Foo is a codeunit and not an interface or the other way around? Would i write code differently or have a better understanding when i see iFoo.Bar()? If the i is really needed, I'd say the thing is poorly named. 
Now, there are cases (and more so in modules) where we have a table, a page, codeunit and an interface, maybe even an enum all with the same name. When you need more than one of those in the same method naming becomes tricky. But even then I'd still not go for the i but prefer a more explicit naming of the use of each of these... 

This is a matter of taste and convention. Partners are always free to choose which conventions they follow. For MS code I strongly advise against the i prefix in both the object name and the variable name, and that is the prevailing pattern in our code now.

Reply:
I agree with "don't do things because they do them", but they have a reason to do so.

Because interfaces are different from concrete objects and can represent many other objects, I find it useful to see that at a glance when reading code. Marking it with an "I" tells me immediately that I'm dealing with a blueprint and not the real implementation. I believe that's why other languages have this naming convention and why it also makes sense for AL.

So, let's agree to disagree here. I'm also strongly against using spaces and slashes or dots in object names. But that's another story. Let's not hijack this thread for a discussion about naming conventions.